RELEASE_NOTES.rst

```diff
@@ -21,6 +21,117 @@
 
 .. towncrier release notes start
 
+Airflow 2.7.2 (2023-10-09)
+--------------------------
+
+Significant Changes
+^^^^^^^^^^^^^^^^^^^
+
+No significant changes
+
+
+Bug Fixes
+"""""""""
+- Import ``AUTH_REMOTE_USER`` from FAB in WSGI middleware example (#34721)
+- Fixed rows count in the migration script (#34348)
```

Comment:
We can replace this with a more informative note user added in the newsfragment

`Fixed AttributeError: 'Select' object has no attribute 'count' during the airflow db migrate command`